lightbeam/send.py

```diff
@@ -78,7 +70,12 @@ def send(self):
             os.makedirs(os.path.dirname(self.lightbeam.results_file), exist_ok=True)
             
             with open(self.lightbeam.results_file, 'w') as fp:
-                fp.write(json.dumps(self.metadata, indent=4))
+                content = json.dumps(self.metadata, indent=4)
+                # failures.line_numbers are split each on their own line; here we remove those line breaks
+                def repl(m):
```

Comment:
We should move this function define outside the any loops or context managers as a minor performance improvement.

Reply:
Excellent point, done.